mlir/test/Target/LLVMIR/llvmir.mlir

```diff
@@ -2621,6 +2621,35 @@ llvm.func @willreturn_call() {
 // CHECK: #[[ATTRS]]
 // CHECK-SAME: willreturn
 
+// -----
+
+llvm.func @f()
+
+// CHECK-LABEL: @no_inline_call
+// CHECK-SAME: #[[ATTRS:[0-9]+]]
```

Comment:
This seems incorrect. This should check for the call operation, not the function, or am I missing something?